app/controllers/v1/users/mutual_friends_controller.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class V1::Users::MutualFriendsController < ApplicationController
+  before_action :pundit_user
+
+  def index
+    page = params[:page] || '1'
```

Comment:
Might be extracted to the application controller method, probably you need it in all controllers.

BTW: what do you use for paging? it should be covered by the gem, I think.

Reply:
Created a `set_page` private method in `application_controller.rb` for this now. And to be explicit in `mutual_friends_controller` (as well as in `users/friends_controller.rb`) I am using it like this:

```ruby
before_action do
    @page = set_page
end

def index
  @user = User.find_by(username: params[:user_username])
  @mutual_friends = @current_user.mutual_friends_with(@user, @page)
  render :mutual_friends, status: :ok
end
```

I'm not using any gem for pagination. Im just doing it manually (as you saw in the `User` model the lines of code where I used `offset` and `limit`. It's a simple feature to implement so I decided to just do it manually :)